src/useSubscription/useSubscription.ts

```diff
@@ -9,7 +9,33 @@ import { uniqueValueWatcher } from '@/utilities/uniqueValueWatcher'
 
 const defaultManager = new Manager()
 
-export function useSubscription<T extends Action>(...[action, args, optionsArg = {}]: SubscribeArguments<T>): UseSubscription<T> {
+/**
+ * The `useSubscription` composition manages data sharing across components. Multiple components can subscribe to an `action` (any method or function) and share the response value.
```

Comment:
From the README minus the top heading and the params/return value tables at the end. 

Considered dropping the README in favor of these more closely co-located documentation bits for easier long-term maintenance, but left it in for now and am using it as a `@see` link both here and for `useSubscriptionWithDependencies` since symbol linking only works if the symbol is imported into the current module so linking to the static readme file is a tad easier.